src/qubex/experiment/protocol/calibration_protocol.py

```
@@ -288,7 +288,7 @@ def calibrate_drag_hpi_pulse(
         n_turns: int = 1,
         n_iterations: int = 2,
         calibrate_beta: bool = True,
-        beta_range: ArrayLike = np.linspace(-0.5, 1.5, 41),
+        beta_range: ArrayLike = np.linspace(-1.5, 1.5, 41),
```

Ruff (0.8.2)

Do not perform function call `np.linspace` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)